containerized-tests.groovy

```diff
@@ -137,10 +137,19 @@ node('cico-workspace') {
 						script: 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} \'source /opt/build/go/src/github.com/ceph/ceph-csi/build.env && echo ${BASE_IMAGE}\'',
 						returnStdout: true
 					).trim()
+					def quay = "quay.io"
+					def docker = "docker.io"
+
+					if (base_image.startsWith(quay)){
```

Comment:
indention is off...

This most likely needs to be changed in other groovy files too.